torchao/quantization/prototype/qat.py

```diff
@@ -25,7 +25,10 @@
     ZeroPointDomain,
 )
 from torchao.quantization.unified import TwoStepQuantizer
-from torchao.quantization.utils import get_group_qparams_symmetric
+from torchao.quantization.utils import (
+    _get_per_token_block_size,
```

Comment:
if it's helpful we could have a general util like:

```
def get_block_size(granularity, *params) -> Callable:
    if granularity == Granularity.PER_BLOCK:
        ...
    elif type == Granularity.PER_TOKEN:
        ...
     ...


block_size = get_block_size(Granularity.PER_TOKEN, 32)(x)
```

Reply:
Sounds good, let's do that separately